db/db.js

```diff
@@ -826,6 +836,53 @@ async function getJobForCourseOfferingId(courseOfferingId) {
     });
 }
 
+async function addLogs(userId, courseOfferingId, action, item, time, json) {
+    return await Log.create({ userId: userId, courseOfferingId: courseOfferingId, action: action, item: item, json: json, createdAt: time });
+}
+
+async function getCoursesByCourseOfferingId(courseOfferingId) {
+  var joinedCourseOfferingTable = 
```

Comment:
Can you write a small comment explaining what this query does?

Reply:
I will push the following comment to my branch also:

**`joinedCourseOfferingTable`**:
Since we need to extract `Terms`, `Depts`, `Universities` information from each table, we combine these tables based on various Ids from `Offering`, then using `OfferingId`s from `CourseOffering` table to map relavent information to each precise `courseOfferings`. This idea is also applied to the intuition of join `Courses` by courseId.

**`joinedUserOfferingTable`**:
Since we need to extract similar information from `Role`, `User`, we combine those tables based on provided Ids from `UserOfferings`. 

**`query`**:
Given a list of `courseOfferingId`s, we want to get a list of detailed information about the course and instructor. Thus, we combine **`joinedUserOfferingTable`** and **`joinedCourseOfferingTable`** create a table that provides every information that we may need. Then we select rows from the pool based on each courseOfferingId and return a list of watchable courses.